pulsar-client-cpp/lib/ClientConnection.cc

```diff
@@ -117,9 +118,6 @@ static Result getResult(ServerError serverError) {
 
         case InvalidTxnStatus:
             return ResultInvalidTxnStatusError;
-
```

Comment:
The changes in this file seems removed some of the latest change in master. This will cause regressions.